### PR TITLE
{Fix} - Missing data and add small tips for angular dev

### PR DESCRIPTION
**Pull request by jenaye**

### CODE REVIEW

File: content/blog/front-end-tips.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+In case you want to contribute on the client part of the application it is useless to launch the codimd or the bitwarden, you just have to launch the API.
+
+
+you can do it this way `docker-compose up api`
+
+ 
+# How to run Angular
+
+For the Angular part, you have to go to the `Client` folder and launch the following commands:
```
Comment (Darkweak) [inline content/blog/front-end-tips.md:23]:
```suggestion
As running javascript in a container is really slow we chose to not provide a container, then you have to go to the `client` folder and run the following command: 
```

File: content/blog/front-end-tips.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
++++
+
+# How tp run API to work correctly
+
+>requirements : node & docker + docker-compose
+
+In case you want to contribute on the client part of the application it is useless to launch the codimd or the bitwarden, you just have to launch the API.
+
+
+you can do it this way `docker-compose up api`
```
Comment (Darkweak) [inline content/blog/front-end-tips.md:18]:
```suggestion
First run `docker-compose up api` to be able to consume the API. Indeed you have to run `make install` first to setup the containers
```

File: data/clients/1.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+name: "customer-1"
+image: "img/clients/customer-1.png"
```
Comment (Darkweak) [inline data/clients/1.yaml:2]: Maybe naming images could be better to replace them as simpler as possible

File: data/testimonials/swift.yaml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+text: "This tool has finally helped me to monitor my teams and the progress of their work. The global vision is really efficient and allows you to see all the important elements at a glance. I love SMERSH and I recommend its use!"
+name: "Swift"
+position: "Consultant security"
+avatar: "img/testimonials/monexcopine.png"
```
Comment (Darkweak) [inline data/testimonials/swift.yaml:4]: WTF

File: data/carousel/design.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+weight: 3
+title: "SMERSH API"
+description: >
+  <ul class="list-style-none">
+    <li>WHICH EXPOSES ALL ROUTES</li><br/>
+    <li>USING CRUD</li>
+  </ul>
+image: "img/carousel/api.png"
```
Comment (Darkweak) [inline data/carousel/design.yaml:8]: Add NL

File: data/features/webdesign.yaml [hunk 6]
```diff
@@ -0,0 +1,5 @@
+weight: 1
+name: "EASY TO DEPLOY"
+icon: "fas fa-drafting-compass"
+url: ""
+description: "All the applications running into docker container"
```
Comment (Darkweak) [inline data/features/webdesign.yaml:5]:
```suggestion
description: "Container first approach for only one configured stack."
```

File: data/features/technos.yaml [hunk 7]
```diff
@@ -0,0 +1,5 @@
+weight: 6
+name: "NEW TECHNOLOGIES"
+icon: "fas fa-cogs"
+url: ""
+description: "Using Symfony 5, Angular 10, Mercure, Vulcain"
```
Comment (Darkweak) [inline data/features/technos.yaml:5]:
```suggestion
description: "Using Symfony 5, Angular 10, Mercure, Vulcain."
```

**Review by darkweak — APPROVED** [review]

**Pull request merged**